lib/middleware/token.js

```diff
@@ -52,15 +54,15 @@ var TokenError = require('../errors/tokenerror');
  */
 module.exports = function token(server, options) {
   options = options || {};
-  
-  if (!server) { throw new TypeError('oauth2orize.token middleware requires a server argument'); }
-  
+
+  if (!server) { throw new TypeError(g.f('{{oauth2orize.token}} middleware requires a server argument')); }
```

Comment:
I don't speak French so I can't comment much on the grammatical correctness to both of those examples. What I can say is I assume the translator will be taking into account the words we omit and make the proper sentences. Meaning they shouldn't translate like Google Translate, but actually read the English sentence and convert it to make sense in the context of that language.